pagopa/pn-frontend
# Pg login - feat/pn-1819

## Short description
Now it is possible to login with a pg user

## List of changes proposed in this pull request
- Updated user model
- Updated user validation
- Managed user roles

## How to test
Go to pa (dev) -> login with baldassaremazza -> choose "Agenzia delle Entrate" -> select "Piattaforma Notifiche PG DEV" -> check that authentication works
